src/MIDebugEngine/AD7.Impl/AD7Module.cs

```diff
@@ -71,6 +73,17 @@ int IDebugModule2.GetInfo(enum_MODULE_INFO_FIELDS dwFields, MODULE_INFO[] infoAr
                     info.m_dwLoadOrder = this.DebuggedModule.GetLoadOrder();
                     info.dwValidFields |= enum_MODULE_INFO_FIELDS.MIF_LOADORDER;
                 }
+                if ((dwFields & enum_MODULE_INFO_FIELDS.MIF_TIMESTAMP) != 0)
```

Comment:
>) [](start = 75, length = 1)

I don't remember the best way to check for this anymore, but you probably only want to do this if we are local debugging. Since otherwise the path that you have may be from another computer. Something like:

Look at the launch options and see if they are LocalLaunchOptions, then do `string.IsNullOrWhiteSpace(localLaunchOptions.MIDebuggerServerAddress) && string.IsNullOrWhiteSpace(localLaunchOptions.DebugServer)`